server.js

```diff
@@ -219,3 +321,23 @@ function Movies(movies) {
   this.image_url = movies.poster_path;
   this.overview = movies.overview;
 }
+
+function Meetups(meetups) {
+  this.name = meetups.events.name;
```

Comment:
add this.tableName = 'trails';

This is to add the tableName to every instance.